src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -275,12 +277,12 @@ const ActiveState = ({
                     case FEED_ITEM_TYPE_VERSION:
                         return (
                             <ActivityItem key={item.type + item.id} className="bcs-version-item" data-testid="version">
-                                {item.versions ? (
+                                {hasMultipleVersions(item) ? (
                                     // $FlowFixMe
-                                    <CollapsedVersion {...item} onInfo={onVersionInfo} />
+                                    <CollapsedVersion {...item} onInfo={onVersionInfo} shouldUseUAA={shouldUseUAA} />
```

Comment:
Do we need the `shouldUseUAA` prop here? Could this also be handled in the `onVersionInfo` function that is passed to `onInfo`?

Reply:
Could be, but this will be removed once the FF is gone, so the parameters passed in can stay in the CollapsedVersion component.